components/upload/chunk-upload.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Chunk Upload
+page_title: Chunk Upload
+description: Enable and configure chunk upload in Upload for Blazor.
+slug: upload-chunk
+tags: telerik,blazor,upload,chunk
+published: True
+position: 3
+---
+
+# Chunk Upload
+
+Chunk upload enables file uploads by dividing the file into smaller parts (chunks) and sending them in multiple requests to the remote endpoint, where they are reassembled into the final file.
+
+## Basics
+
+To setup the feature, use the `UploadChunkSettings` tag, which is nested inside `UploadSettings`. The tag includes the following parameters:
+
+| Parameter | Type and Default Value | Description |
+|----------|----------|----------|
+| `AutoRetryAfter` | `double` <br/> (100) | Specifies the amount of time in milliseconds after which a failed chunk upload request will be retried.  
+| `Enabled` | `bool` <br/> (`true`) | Specifies if the chunk upload is enabled.  
+| `MaxAutoRetries` | `int` <br/> (1) | Specifies the number of attempts to retry uploading a failed chunk.  
+| `MetadataField` | `string` <br/> (`chunkMetadata`) | Specifies the name of the variable that will receive the chunk upload metadata in the remote endpoint.  
+| `Resumable` | `bool` <br/> (`true`) | Specifies if the file upload process could be paused and later resumed.  
```

Comment:
To me, it is not clear what can pause the process. Maybe we can add some more details here?